lib/applb/cli.rb

```diff
@@ -2,6 +2,9 @@
 require 'optparse'
 require 'pathname'
 
+require 'aws-sdk-core'
+Aws.use_bundled_cert!
```

Comment:
Thanks for the PR!

I don't think use_bundled_cert is good to defined in gem.
If people wanted to use bundled cert, they can in other ways.

Please remove this line, or do you have any specific reason to add this line?

Reply:
I understand.
However, this line written in not only this repo but also miam. (https://github.com/codenize-tools/miam/blob/0.2.4/lib/miam.rb#L9)

I think that if this line is removed, the miam code should also be removed, is this right?

Reply:
Yes, I think that it is a good point.
But in other words, **only** miam uses `Aws.use_bundled_cert!` in codenized-tools.